### DISCUSSION TITLE
throw redirect() in loader and action

### DISCUSSION THREAD

**Discussion by harshmangalam** [sveltejs/kit#7712] [Q&A] [answered]

**Comment by teemingc** [comment, marked as answer]
Sorry, could you rephrase your question?
Edit: Here are some possibly useful responses about `throw redirect()`:
* [my thoughts about it](https://github.com/sveltejs/kit/discussions/6693#discussioncomment-3606878)
* [comment from dummidum (official maintainer)](https://github.com/sveltejs/kit/discussions/6693#discussioncomment-3731436)
